proto/eth/evm/v1/query.proto

```diff
@@ -76,8 +76,8 @@ service Query {
     option (google.api.http).get = "/nibiru/evm/v1/base_fee";
   }
 
-  rpc TokenMapping(QueryTokenMappingRequest) returns (QueryTokenMappingResponse) {
-    option (google.api.http).get = "/nibiru/evm/v1/token_mapping/{token}";
+  rpc FunToken(QueryFunTokenRequest) returns (QueryFunTokenResponse) {
```

Comment:
(nit) I kind of liked the `TokenMapping` service name. As an end user, does `FunToken` make more sense or `TokenMapping` make more sense, or maybe a third alternative `FungibleTokenMapping`.

Reply:
(nit) It's still not too late to change `FunToken` to `FunTokenMapping` in other parts of the codebase.